Server/src/routes/poitype.route.ts

```diff
@@ -84,7 +84,7 @@ export class PoiTypeRoute {
 	private async createType(req: Request, res: Response): Promise<void> {
 		const { name, icon, description }: CreatePOIType = req.body
 
-		const poiType: POIType | null = await database.pois.saveType(name, icon, description)
+		const poiType: POIType | null = await database.pois.saveType({ name, icon, description })
```

Comment:
I just double checked this: It is indeed possible!

In my first iteration for this branched I just changed the database calls in the routes and services so that the methods are still working like intended but we can change it here to parse the `poiTypeInput` directly